sdk/python/tests/compiler/testdata/custom_task_spec_customtask_cr1.yaml

```diff
@@ -1,20 +0,0 @@
-# Copyright 2021 kubeflow.org
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-apiVersion: custom_task_api_version
```

Comment:
Just quick question, isn't `custom_task_ref.py ` will generate this CRD since we are using taskRef?

Reply:
It is generated in `tmp/` dir and cleaned up. Our tests do not assert the CRDs. Do you think it should be changed?

Reply:
I opened an issue on it, but we didn't get time to implement it yet. 
https://github.com/kubeflow/kfp-tekton/issues/610